docs/src/user-guide/core-unstructured/clp.md

```diff
@@ -105,25 +105,36 @@ Currently, timestamps must be specified as milliseconds since the UNIX epoch.
 
 # Parallel Compression
 
-By default, `clp` uses an embedded SQLite database, so each directory containing archives can only
-be accessed by a single `clp` instance.
+To enable parallel compression to the same archives directory, `clp` (and by extension, `clg`) needs
+to be configured to use a MySQL-type database (e.g., MariaDB) rather than the default---an embedded
```

Comment:
```suggestion
to be configured to use a MySQL-compatible database (e.g., MariaDB) rather than the default---an embedded
```
or
```suggestion
to be configured to use a MySQL-compatible RDBMS (e.g., MariaDB) rather than the default---an embedded
```